Source/Csla.Blazor.WebAssembly.Tests/SessionManagerTests.cs

```diff
@@ -0,0 +1,204 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Csla.Blazor.WebAssembly.State;
+using Csla.State;
+using Moq;
+using System.Net.Http;
+using Csla.Blazor.WebAssembly.Configuration;
+using Csla.Core;
+using Csla.Runtime;
+using Moq.Protected;
+using System.Net;
+using Csla.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+using Csla.Serialization.Mobile;
+using Microsoft.AspNetCore.Components.Authorization;
+
+namespace Csla.Test.State
+{
+  [TestClass]
+  public class SessionManagerTests
+  {
+    private SessionManager _sessionManager;
+    private SessionMessage SessionValue;
```

Comment:
Field with `_s` not PascalCase.

Reply:
I think you missunderstood me.
`_sessionManager` is correct.
`SessionValue` is not. It should be `_sessionValue`.